src/device/wg_controller.rs

```diff
@@ -194,10 +194,18 @@ async fn consolidate_wg_peers<
                 if let (Some(sk), Some(mesh_ip)) =
                     (session_keeper, requested_peer.peer.allowed_ips.first())
                 {
+                    // TODO this should be acommodatedto IPv6
```

Comment:
Nvm I see it's not the main MR for ipv6 support. What is the reason for this specific split?

Reply:
Yeah, so this was the red line, where I decided to draw between ipv6 support in `session_keeper` and ipv6 support in `wireguard`. This will accommodate the ipv6 address handling in still-in-progress [PR](https://github.com/NordSecurity/libtelio/pull/61).